docs/workforce-management-framework/topics/pro-Using-Keycloak-implementation.adoc

```diff
@@ -0,0 +1,89 @@
+[id='enable-keycloak-integration-{chapter}']
+= Enable {WFM-RC-KeycloakName} Integration
+
+The {WFM-RC-NameLong} Demo Applications allow for Authentication and Access Management using
+link:http://www.keycloak.org/index.html[{WFM-RC-KeycloakName}].
+The {WFM-RC-NameLong} {WFM-RC-KeycloakName} integration is situated inside of the Demo Applications.
+
+By default, the {WFM-RC-KeycloakName} Authentication and Access Management is not setup with the Demo Applications.
+A {WFM-RC-NameLong} {WFM-RC-KeycloakName} Docker image has been created to allow easy deployment of a {WFM-RC-KeycloakName} server configured
+with the {WFM-RC-NameLong} demo data.
+
+== Run {WFM-RC-KeycloakName} Docker image
+
+The image is published to docker hub
+
+    docker run -p 8080:8080 feedhenry/raincatcher-keycloak
+
+For more information on this docker image see the docker image link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/README.md[README.md].
+
+NOTE: If you wish to configure your own instance of {WFM-RC-KeycloakName} see the following
+link:{WFM-RC-KeycloakURL}getting_started/topics/first-boot.html[getting started guide]. You will need to import our Realm
+settings by importing link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[/data_files/raincatcher-realm.json].
```

Comment:
"settings by importing link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[/data_files/raincatcher-realm.json]."

**>**

"settings by importing link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[/data_files/raincatcher-realm.json]. `file if you would like to use our raincatcher demo realm in your Keycloak instance.`"

Small clarification. (The first sentence says to go create your own keycloak instance, the second sentence says you must import our realm - _only if you want the raincatcher demo realm in your instance)

Reply:
Reworded
"NOTE: If you wish to configure your own instance of {WFM-RC-KeycloakName} see the following
link:{WFM-RC-KeycloakURL}getting_started/topics/first-boot.html[getting started guide]. If you wish to use the demo application will need to import our Realm
settings by importing link:{WFM-RC-KeycloakDockerURL}{WFM-RC-Branch}/data_files/raincatcher-realm.json[/data_files/raincatcher-realm.json] file."